integration-tests/__tests__/__snapshots__/empty-describe-with-hooks.test.js.snap

```diff
@@ -0,0 +1,142 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`hook in empty describe 1`] = `
+Object {
+  "rest": "FAIL __tests__/hook-in-empty-describe.test.js
+
+
+  ● Test suite failed to run
+
+    thrown: \\"Invalid: beforeEach() in a describe block with no tests.\\"
```

Comment:
@SimenB This is using the same formatting as user errors. I'm not sure if these errors should have the `thrown: ` prefix. Thoughts?

Also, not sure about the wording here. Got any suggestions for other wording that would be more in-line with other Jest errors?

Reply:
Yeah, the `thrown` part is a bit heavy handed. Not sure how to bes differentiate. Maybe you can just copy the stack from `asyncError` and append your own message? Like `state.unhandledErrors.push({message: myMessage: stack: asynError.stack})` or something?

Reply:
The "thrown" is actually added in `_formatError`: https://github.com/facebook/jest/blob/master/packages/jest-circus/src/utils.js#L315

Reply:
You can probably just do `asyncError.message = 'my message'` and push it into errors, and it'll be correct